### ISSUE TITLE
Error in get_juicer_metadata()

### ISSUE THREAD

**Issue by teunbrand** [robinweide/GENOVA#152]
Calling load_contacts in the following way:

```
exp <- load_contacts("../../hic_roa_2017/GSM2795535_Rao-2017-HIC001.hic",
                     resolution = 10e3, sample_name = "HCT116",
                     colour = "limegreen")
```

Yielded the following error:

> null terminator not found: breaking string at 10000 bytes
> Warning in for (i in seq_along(x)) { :
  closing unused connection 5 (../../hic_roa_2017/GSM2795535_Rao-2017-HIC001.hic)

I've traced the steps until where errors occur and the errors occur in lines 114-118 of the juicerUtils.R script:

```
  chrom_name_length = lapply(1:n_chrs, function(x){
    chrom = readBin(file2read, character(), size = 1)
    len = readBin(file2read, integer(), n = 1)
    data.table::data.table(chrom, len)
  })
```

In my case, `n_chrs` was evaluated previously to be `1952543859L`, which I would suppose is quite a large number of chromosomes to loop over.

**Comment by teunbrand** [comment]
I think it might be related to the attributes, as that particular file has 2 attributes and as far as I can interpret binary file parsing, there does not seem to be anything related to attribute parsing.

**Comment by robinweide** [comment]
I think the first thing to tackle is the high number of n_chrs.  This is due to not handling the attributes correctly. Never saw such a file, so let's look at this tomorrow.

**Comment by teunbrand** [comment]
I think this is now fixed along with #166
